### ISSUE TITLE
[Bug Report] Do not show the name of the songs

### ISSUE THREAD

**Issue by ancar0** [jagrosh/MusicBot#500]
**Describe the bug**
Doesn't show the name of the songs.

**To Reproduce**
Play any song and it won't show any name of any song.

**Expected behavior**
Show the name of the song.

**Screenshots**
![image](https://user-images.githubusercontent.com/49865848/94054099-14603e80-fdd3-11ea-8a62-94b2a7387f7c.png)
![image](https://user-images.githubusercontent.com/49865848/94054174-335ed080-fdd3-11ea-9594-20fb72d56f28.png)

**Debug Info**
```
System Properties:
  java.version = 11.0.8
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 11
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 11.0.8+10
  java.specification.version = 11
  os.arch = amd64
  os.name = Windows 10

JMusicBot Information:
  Version = 0.2.10
  Owner = 187667640385273866
  Prefix = <
  AltPrefix = null
  MaxSeconds = 0
  NPImages = false
  SongInStatus = false
  StayInChannel = false
  UseEval = false
  UpdateAlerts = true

Dependency Information:
  JDA Version = 3.8.3_464
  JDA-Utilities Version = 2.1.5
  Lavaplayer Version = 1.3.50

Runtime Information:
  Total Memory = 96
  Used Memory = 40

Discord Information:
  ID = 710693314558558237
  Guilds = 12
  Users = 602
```

**Comment by ancar0** [comment]
![image](https://user-images.githubusercontent.com/49865848/94054616-c566d900-fdd3-11ea-8fa9-6737e2dd97f9.png)

**Comment by Amiiya-san** [comment]
I can confirm I get the same unexpected behiavour. 
```
System Properties:
  java.version = 1.8.0_265
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 1.8
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 1.8.0_265-8u265-b01-0+deb9u1-b01
  java.specification.version = 1.8
  os.arch = amd64
  os.name = Linux

JMusicBot Information:
  Version = 0.2.10
  Owner = 278230133176008704
  Prefix = //
  AltPrefix = ..
  MaxSeconds = 0
  NPImages = false
  SongInStatus = false
  StayInChannel = false
  UseEval = false
  UpdateAlerts = true

Dependency Information:
  JDA Version = 3.8.3_464
  JDA-Utilities Version = 2.1.5
  Lavaplayer Version = 1.3.50

Runtime Information:
  Total Memory = 556
  Used Memory = 413

Discord Information:
  ID = 512030715332919348
  Guilds = 94
  Users = 145217```

However, I am getting a different error:
java.lang.RuntimeException: No player info block, at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.load(DefaultYoutubeTrackDetailsLoader.java:62)

**Comment by Amiiya-san** [comment]
When testing, I found out that this is only related to YouTube playlists.
When loading songs via their link or by their name, they show up normally.
![issue image](https://i.iya.at/chrome_SGcQZu.png)

**Comment by ancar0** [comment]
Then why only playlists don't show the name.
This never happened before. Why now and why only playlists. I have 2 bots of these, Both having the same issue.
Meaning it's something about youtube.

**Comment by Proxymiity** [comment]
The YouTube API might have changed, and Lavaplayer devs hasn't fixed the issue yet I think.

**Comment by Sanduhr32** [comment]
Correct @Proxymiity

**Comment by ancar0** [comment]
Ok. Thank you @Proxymiity 
@Sanduhr32 When it will be fixed?

**Comment by Sanduhr32** [comment]
@ancar0 if i would know this, I would have said it. However I don't know how well maintained Lavaplayer is right now. It depends on Lavaplayer since months to get fixes.

**Comment by Proxymiity** [comment]
I think this issue may be related to what we are experiencing: https://github.com/sedmelluq/lavaplayer/issues/525

**Comment by ThaddeusKKR** [comment]
Even after using #501, I still get this issue.
![image](https://user-images.githubusercontent.com/49682825/94362307-f7867c80-00ec-11eb-8d20-76afbcaed847.png)
Debug:
```
System Properties:
  java.version = 1.8.0_262-heroku
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 1.8
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 1.8.0_262-heroku-b10
  java.specification.version = 1.8
  os.arch = amd64
  os.name = Linux

JMusicBot Information:
  Version = 0.2.10
  Owner = 275830234262142978
  Prefix = sy.
  AltPrefix = @mention
  MaxSeconds = 0
  NPImages = false
  SongInStatus = false
  StayInChannel = true
  UseEval = true
  UpdateAlerts = true

Dependency Information:
  JDA Version = 3.8.3_464
  JDA-Utilities Version = 2.1.5
  Lavaplayer Version = 1.3.50

Runtime Information:
  Total Memory = 239
  Used Memory = 85

Discord Information:
  ID = 709391431713685580
  Guilds = 9
  Users = 91159
```

**Comment by Sanduhr32** [comment]
@ThaddeusKKR #501 is not merged so there is no way that you used it's changes yet.

**Comment by Proxymiity** [comment]
@Sanduhr32 Did you try to compile with #501?

**Comment by Sanduhr32** [comment]
@Proxymiity self-building isn't supported, so i don't provide official jars of @jagrosh project. I can confirm that my self-built jar works tho. I also highly doubt that they built my fork since it would have displayed `1.3.52` instead of `1.3.50` for Lavaplayers version.

**Comment by Proxymiity** [comment]
@Sanduhr32 Sure, I just wanted to know if your patch fixes this issue, not to distribute it :)

**Comment by Sanduhr32** [comment]
![grafic](https://user-images.githubusercontent.com/25187502/94366329-cb143580-00d7-11eb-905f-8b4277ad6007.png)
@Proxymiity 
```
System Properties:
  java.version = 14.0.2
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 14
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 14.0.2+12
  java.specification.version = 14
  os.arch = amd64
  os.name = Linux

JMusicBot Information:
  Version = Snapshot
  Owner = 198137282018934784
  Prefix = @mention
  AltPrefix = &&
  MaxSeconds = 0
  NPImages = true
  SongInStatus = false
  StayInChannel = true
  UseEval = true
  UpdateAlerts = true

Dependency Information:
  JDA Version = 3.8.3_464
  JDA-Utilities Version = 2.1.5
  Lavaplayer Version = 1.3.51

Runtime Information:
  Total Memory = 80
  Used Memory = 74

Discord Information:
  ID = 272319506951700480
  Guilds = 6
  Users = 4069
```

**Comment by nightcat91** [comment]
I had the same issue described here...
So when can we expect this to be fixed? since there seems to be a self compiled fix already?

**Comment by Proxymiity** [comment]
It will be fixed when the project's maintainer will look at this issue I think (or someone with write access to the repo). Currently, you need to wait a fix.

**Comment by jagrosh** [comment]
This should be solved as of version 0.3.1
